testData/solutions/web.json

```diff
@@ -164,5 +164,131 @@
+                                }
+                            }
+                        }
+                    },
+                    "links": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "inputPath": "http://registry\\.gpii\\.net/common/linkAdaptation",
+                            "options": {
+                                "emphasizeLinks": {
+                                    "outputValue": true
+                                }
+                            }
+                        }
+                    },
+                    "contrastTheme": {
+                        "transform": {
+                            "type": "fluid.transforms.condition",
+                            "conditionPath": "http://registry\\.gpii\\.net/common/highContrastEnabled",
+                            "true": {
+                                "transform": {
+                                    "type": "fluid.transforms.valueMapper",
+                                    "inputPath": "http://registry\\.gpii\\.net/common/highContrastTheme",
+                                    "defaultInputValue": "default",
+                                    "options": {
+                                        "black-white": {
+                                            "outputValue": "bw"
+                                        },
+                                        "white-black": {
+                                            "outputValue": "wb"
+                                        },
+                                        "black-yellow": {
+                                            "outputValue": "by"
+                                        },
+                                        "yellow-black": {
+                                            "outputValue": "yb"
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "signLanguageEnabled": "http://registry\\.gpii\\.net/common/signLanguageEnabled",
+                    "signLanguage": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "inputPath": "http://registry\\.gpii\\.net/common/signLanguage",
+                            "options": {
+                                "ase": {
+                                    "outputValue": "ase"
+                                },
+                                "gsg": {
+                                    "outputValue": "gsg"
+                                },
+                                "ils": {
+                                    "outputValue": "ils"
+                                }
+                            }
+                        }
+                    },
```

Comment:
I have now added 
  "defaultInputValue": "ils",
to the valueMapper.

(I found this feature in several transformations for Linux, e.g. https://github.com/GPII/universal/blob/master/testData/solutions/linux.json#L40.)

However, this has an unintended side effect: the transformer now outputs "ils" when the preference signLanguage (as a common term) is not present in a preference set. Is this the intended behaviour of this valueMapper feature? I would expect no output when the preference is not expressed in the NP set. 
I could not find this documented at http://wiki.gpii.net/w/Architecture_-_Available_transformation_functions#Using_the_valueMapper_to_convert_to.2Ffrom_enumerated_sets_of_values.